frontend/src/app/modules/intervention-form/service/intervention-form.service.ts

```diff
@@ -0,0 +1,25 @@
+import { Injectable } from '@angular/core';
+import { HttpClient, HttpHeaders } from '@angular/common/http';
+
+import { Observable } from 'rxjs';
+
+import { InterventionFormData, InterventionPostData } from '@interventionForm/types';
+import { environment } from '@environment';
+
+const HTTP_OPTIONS = {
+  headers: new HttpHeaders({
+    // TODO: We need to find a way to securely pass access token for Azure Functions
```

Comment:
As for local-dev token, it is just a string that exists in a file - we don't have an endpoint to retrieve it, so I came up with a solution to read it from file that will be ignored in .gitignore - to prevent any possibility of submission. I have already written code that does exactly this, using Angular's HttpInterceptors (great feature btw). I will have it ready for review in the near time, in the next PR.

As for real production usage, I guess we need to first understand the exact details of authorization on Azure, etc. but my wild guess is that we should store the token in local storage or a cookie - just as you mentioned. And we will need to write that part as well ;)

Reply:
Cool! :)